README.md

```diff
@@ -476,6 +477,7 @@ When learning CS, there are some useful sites you must know to get always inform
 - [KudVenkat](https://www.youtube.com/user/kudvenkat/) : Microsoft .NET technologies and SQL Server tutorials.
 - [The Net Ninja](https://www.youtube.com/channel/UCW5YeuERMmlnqo4oq8vwUpg): Web development tutorials
 - [Web Dev Simplified](https://www.youtube.com/channel/UCFbNIlppjAuEX4znoulh0Cw): Tutorials based on Web development from basic to advanced.
+- [Wes Bos](https://www.youtube.com/user/wesbos): Youtube channel for Wes Bos. Has various coding tutorials and free courses. 
```

Comment:
We are deprecating adding of youtube channels. Can you please update and leave only interviewcake link? Thanks